### PR TITLE
Use psycopg3 instead of psycopg2

### PR DESCRIPTION
**Pull request by lospejos**

### CODE REVIEW

File: pg_analyse/toolbox.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from typing import List, Union, Set, Dict
 
-import psycopg2
+import psycopg
```
Comment (idlesign) [inline pg_analyse/toolbox.py]: Let's make it in a backward compatible manner for those who has no chance to upgrade

```
try:
    import psycopg

except ImportError:
    import psycopg2 as psycopg

```

Here and in setup.py it seems.

File: README.rst [hunk 2]
```diff
@@ -44,15 +44,15 @@ Requirements
...
 Installation
 ------------
 
 .. code-block:: bash
 
-    ; If you do not have psycopg2 yet, install it as `psycopg2` or `psycopg2-binary`. 
+    ; If you do not have psycopg3 yet, install it as `psycopg` or `psycopg[binary]`.
     ; You may also want to install `envbox` to get PG connection settings from .env files.
     ; Your distribution may require issuing `pip3` command instead of plain `pip`.
     $ pip install psycopg2-binary envbox
```
Comment (idlesign) [inline README.rst]: Old command is used.

**Review by idlesign — CHANGES_REQUESTED** [review]

**Comment by idlesign** [comment]
Thank you. Have some hints though.

File: pg_analyse/toolbox.py (hunk 1 above)
Comment (lospejos) [reply pg_analyse/toolbox.py]: Updated my PR according to your comment, please check

File: README.rst (hunk 2 above)
Comment (lospejos) [reply README.rst]: Didn't get what exactly has to be changed here, honestly. Could you give an example?

Comment (idlesign) [reply README.rst]: I'd expect something like `pip install psycopg[binary] envbox`

Comment (lospejos) [reply README.rst]: You was right, fixed this in new commit

**Comment by idlesign** [comment]
Thank you! Merged.

**Pull request merged**